src/Content/Version.php

```diff
@@ -76,18 +85,25 @@ public function delete(string|null $language = null): void
 	 */
 	public function ensure(
 		string $language = 'default'
-	): void {
+	): bool {
 		if ($this->exists($language) !== true) {
-			throw new NotFoundException('Version "' . $this->id . ' (' . $language . ')" does not already exist');
+			$message = match($this->model->kirby()->multilang()) {
+				true  => 'Version "' . $this->id . ' (' . $language . ')" does not already exist',
+				false => 'Version "' . $this->id . '" does not already exist',
+			};
+
+			throw new NotFoundException($message);
 		}
+
+		return true;
```

Comment:
`Question`: What is the plan for this return value? In its current form the method only ever returns `true` or throws an exception, so the return value becomes meaningless. Are there plans to add a `false` return later?

Reply:
No, it could really also be marked as void and return nothing instead. It's more of a personal preference. I read it like "Everything fine down here" but from a usage perspective, it does not add any value.

Reply:
`Suggestion`: I switched to an early return in the method. For me, this is an even better argument to return true here, for readability reasons. Returning void would still work as well though. I think both ways are fine for me.

Reply:
I'd prefer `void`. `bool` suggests that there could be `false` as well, which isn't the case here.

Reply:
Ok, I've refactored the method to use void as return type.